src/coreclr/nativeaot/System.Private.TypeLoader/src/Internal/Runtime/TypeLoader/TypeBuilder.cs

```diff
@@ -473,37 +459,27 @@ public GCLayout(RuntimeTypeHandle rtth)
                 Debug.Assert(MethodTable != null);
 
                 _bitfield = null;
-                _isReferenceTypeGCLayout = false; // This field is only used for the LowLevelList<bool> path
                 _gcdesc = MethodTable->ContainsGCPointers ? (void**)MethodTable - 1 : null;
                 _size = (int)MethodTable->BaseSize;
             }
 
             /// <summary>
             /// Writes this layout to the given bitfield.
```

Comment:
There is no "given bitfield" anymore. This comment should be updated.

Also, should the method be renamed to `GetBitField`? It is not necessarily "writing to" anything.